free-1.md

```diff
@@ -58,11 +117,25 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+About 30-40% of the time would be spent engaging with the language aspects, while 60-70% will be working on the systems aspects of this project. 
```

Comment:
I think that this is a pretty large topic idea that might be too ambitious to complete over the course of the semester. However, I think that you could definitely focus and complete the language during the semester and save the systems aspects for a later time.